e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -19,6 +19,10 @@ export function getDashboardCard(index = 0) {
   return getDashboardCards().eq(index);
 }
 
+export function ensureDashboardCardHasText(text, index = 0) {
+  cy.get(".Card").eq(index).findByText(text);
```

Comment:
```suggestion
  cy.get(".Card").eq(index).should("contain", text);
```

A bit more readable, imo.